test/stressRooFit_tests.h

```diff
@@ -1270,12 +1270,12 @@ class TestBasic205 : public RooUnitTest
     RooExponential bkg2("bkg2","Background 2",x,alpha) ;
 
     // Sum the background components into a composite background p.d.f.
-    RooRealVar bkg1frac("sig1frac","fraction of component 1 in background",0.2,0.,1.) ;
-    RooAddPdf bkg("bkg","Signal",RooArgList(bkg1,bkg2),sig1frac) ;
+    RooRealVar bkg1frac("bkg1frac","fraction of component 1 in background",0.8,0.,1.) ;
+    RooAddPdf bkg("bkg", "Signal", {bkg1,bkg2}, bkg1frac);
```

Comment:
Also the title of the pdf should be changed to something like "Total Background"